### PR TITLE
update batch transcription samples with API v3.1 features

### PR DESCRIPTION
**Pull request by chlandsi**
## Purpose

* ...

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: samples/batch/csharp/batchclient/dto/diarizationproperties.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+//
+
+namespace BatchClient
+{
+    using Newtonsoft.Json;
```
Comment (rarera) [inline samples/batch/csharp/batchclient/dto/diarizationproperties.cs:8]: not needed

**Review by rarera — APPROVED** [review]

File: samples/batch/csharp/batchclient/dto/diarizationproperties.cs (hunk 1 above)
Comment (chlandsi) [reply samples/batch/csharp/batchclient/dto/diarizationproperties.cs]: fixed, thanks!

File: samples/batch/python/python-client/main.py [hunk 2]
```diff
@@ -143,25 +143,32 @@ def transcribe():
...
     # for supported parameters.
-    properties = {
-        # "punctuationMode": "DictatedAndAutomatic",
-        # "profanityFilterMode": "Masked",
-        # "wordLevelTimestampsEnabled": True,
-        # "diarizationEnabled": True,
-        # "destinationContainerUrl": "<SAS Uri with at least write (w) permissions for an Azure Storage blob container that results should be written to>",
-        # "timeToLive": "PT1H"
-    }
+    properties = cris_client.TranscriptionProperties()
```
Comment (eric-urban) [inline samples/batch/python/python-client/main.py:146]: Can we take this opportunity to move away from "cris"?

**Comment by eric-urban** [comment]
@chlandsi - Should we wait for GA before merging? We still need to support 3.0 clients.

**Review by zhouwangzw — DISMISSED** [review]

File: samples/batch/python/python-client/main.py (hunk 2 above)
Comment (chlandsi) [reply samples/batch/python/python-client/main.py]: done

File: samples/batch/csharp/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-**C# Example to use the Speech Services Batch Transcription API**
+# C# Example to use the Speech Services Batch Transcription API
 
-Speech Service Batch Transcription is exposed through a REST API. The samples here does **NOT** require the installation of the Cognitive Service Speech SDK, but uses the REST API directly.
+Speech Services Batch Transcription is exposed through a REST API. The samples here does **NOT** require the installation of the Cognitive Service Speech SDK, but uses the REST API directly.
```
Comment (eldakms) [inline samples/batch/csharp/README.md:3]: typo: The sample here does NOT .. should be do NOT

Comment (eldakms) [reply samples/batch/csharp/README.md:3]: also uses -> use

**Review by eldakms — APPROVED** [review]

**Review by zhouwangzw — DISMISSED** [review]

File: samples/batch/csharp/README.md (hunk 3 above)
Comment (chlandsi) [reply samples/batch/csharp/README.md]: fixed, thanks

**Review by zhouwangzw — APPROVED** [review]

**Pull request merged**